packages/mobile/src/services/offline-downloader/offline-storage.ts

```diff
@@ -105,7 +106,8 @@ export const purgeDownloadedCollection = async (collectionId: string) => {
   const collectionDir = getLocalCollectionDir(collectionId)
   if (!(await exists(collectionDir))) return
   await RNFS.unlink(collectionDir)
-  store.dispatch(removeCollection(collectionId))
+  store.dispatch(removeCollection({ collectionId, isFavoritesDownload: true }))
+  store.dispatch(removeCollection({ collectionId, isFavoritesDownload: false }))
```

Comment:
This may create a new entry in either `state.favoritedCollections` or `state.collections` in the offline downloads slice, should be fine but not sure if you intended that

Reply:
yeah that's fine as long as its value is set to false